app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
+}
+
 .input-group > .search-autocomplete-wrapper {
   border: none;
 }
 
-.search-query-form, .navbar-search .search-query-form{
+.input-group > .search-autocomplete-wrapper input {
+  padding-top: 0;
+  padding-bottom: 0
+}
+
+.search-query-form, .navbar-search .search-query-form {
   border: 1px solid #ABABA9;
-  padding: .5rem;
+  padding: 8px;
   border-radius: .25rem;
   background-color: white;
+  max-height: 47px;
+  width: 100%;
+  @media (min-width: 576px) {
+    max-width: 520px;
+  }
 }
 
 .navbar-search {
   --bs-light-rgb: var(--stanford-fog-light-rgb);
 }
 
+.navbar-search > .container-fluid {
+  justify-content: center;
```

Comment:
can this be rewritten as:
```
@media(max-width: 767px) {
  justify-content: center;
 }
 ```
 Then I think you can leave out the `justify-content: space-between`, right?

Reply:
I always default to min-width over max-width for a mobile first approach. Max-width is last resort if you can't do it another way.
I can see in this case with the GBL overrides why this could be a good approach.